packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -310,6 +313,23 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         }
     }
 
+    public async tombstone() {
+        this.verifyNotClosed();
```

Comment:
Why check for not closed here?

Reply:
If the datastore is disposed, you should not be touching it anyway right? Or am I missing something

Reply:
Why not? If a data store is closed for some reason, we should be able to tombstone / delete it. Imagine a data store has some problem and cannot be loaded and is disposed every time. GC should be able to come in and remove it so that the document works fine.
This scenario has actually happened where due to a "failure summary" feature, certain data stores would never load. Since there was no GC, we had to take a lot of manual steps to fix it.